simd.dd

```diff
@@ -57,9 +57,25 @@ import core.simd;
 ---
 
 	$(P These types and operations will be the ones defined for the architecture
-	the compiler is targetting. If a particular CPU family has varying
+	the compiler is targeting. If a particular CPU family has varying
 	support for vector types, an additional runtime check may be necessary.
-	The compiler does not emit runtime checks; those must be done by the programmer.
+	The compiler does not emit runtime checks; those must be done by the 
+	programmer.
+	)
+	
+	$(P It is possible to test if the architecture the compiler is targeting
+	supports SIMD types by using:)
+---
+version(D_SIMD)
```

Comment:
The *correct* way to determine whether or not the architecture you are targeting supports a given vector natively is by using the method here: https://github.com/D-Programming-Language/druntime/blob/master/src/core/simd.d#L41

Of course, if you just want to test if the ``core.simd`` alias exists, then ``__traits(compiles, float4 f)``will work just as well.